addons/binding/org.openhab.binding.modbus/README.md

```diff
@@ -1030,6 +1030,158 @@ There is no trigger anymore but one can use transformations to accomplish the sa
+
+The 1.x modbus configuration defined 4 slaves:
+
+`modbus.cfg`
+
+```
+    poll=500
+
+    tcp.slave1.connection=192.168.2.9:502
+    tcp.slave1.type=coil
+    tcp.slave1.id=1
+    tcp.slave1.start=12288
+    tcp.slave1.length=128
+    tcp.slave1.updateunchangeditems=false
+
+    tcp.slave2.connection=192.168.2.9:502
+    tcp.slave2.type=holding
+    tcp.slave2.id=2
+    tcp.slave2.start=12338
+    tcp.slave2.length=100
+    tcp.slave2.updateunchangeditems=false
+
+    tcp.slave3.connection=192.168.2.9:502
+    tcp.slave3.type=holding
+    tcp.slave3.id=3
+    tcp.slave3.start=12438
+    tcp.slave3.length=100
+    tcp.slave3.updateunchangeditems=false
+
+    tcp.slave4.connection=192.168.2.9:502
+    tcp.slave4.type=holding
+    tcp.slave4.id=4
+    tcp.slave4.start=12538
+    tcp.slave4.length=100
+    tcp.slave4.updateunchangeditems=false
+```
+
+As you can see, all the slaves poll the same modbus device (actually a Wago 750-841 controller). We now have to create ``Things`` for this pollers.
+
+The 2.x modbus binding uses a three-level definition. Level one defines a ``Bridge``for every modbus slave. The 1.x configuration only has one slave, so there will be one top level bridge.
+
+```
+Bridge modbus:tcp:wago [ host="192.168.2.9", port=502, id=1 ] {
+
+}
+```
+Host and Port are taken from the 1.x modbus config.
+
+Within the top level ``Bridge`` there are one or more second level bridges that replace the former slave configurations. The poll frequency can now be set per slave, so you may want to define different poll cycles up to your needs. The slave ``Bridge`` configs go inside the top level config. For the four pollers defined in this example the 2.x configuration looks like this:
+
+```
+Bridge modbus:tcp:wago [ host="192.168.2.9", port=502, id=1 ] {
+
+    Bridge poller wago_slave1 [ start=12288, length=128, refresh=500, type="coil" ] {
+    }
+
+    Bridge poller wago_slave2 [ start=12338, length=100, refresh=4000, type="holding" ] {
+    }
+
+    Bridge poller wago_slave3 [ start=12438, length=100, refresh=5000, type="holding" ] {
+    }
+
+    Bridge poller wago_slave4 [ start=12538, length=100, refresh=10000, type="holding" ] {
+    }
+}
+```
+
+Address, length and type can be directly taken over from the 1.x config.
+
+The third (and most complex) part is the definition of data ``Thing`` objects for every ``Item`` bound to modbus. This definitions go into the corresponding 2nd level ``Bridge`` definitions. Here it is especially important that the modbus binding now uses absolute addresses all over the place, while the addresses in the item definition for the 1.x binding were relative to the start address of the slave definition before. For less work in the following final step, the update of the ``Item`` configuration, the naming of the data things in this example uses the offset of the modbus value within the poller as suffix, starting with 0(!) 
```

Comment:
Please split the lines
Single back ticks

Reply:
I'm not sure I follow this
> For less work in the following final step, the update of the ``Item`` configuration, the naming of the data things in this example uses the offset of the modbus value within the poller as suffix, starting with 0(!)

Reply:
`data` instead of plain data to match documentation elsewhere? (for me it makes it easier to the eye to notice it refers to "code" / configuration keyword)

Reply:
Backticks and linesplit done. The naming scheme makes it easier to update the data file, example given a few lines further below.

`data` changed